### ISSUE TITLE
Error when creating Page-Link with sub-links in navbar

### ISSUE THREAD

**Issue by Sreini** [coderedcorp/coderedcms#533]
#### Describe the bug
I have made a navbar with codered and have attempted to add a page link to it with sub-links. 
![image](https://user-images.githubusercontent.com/36940685/196981315-13963c51-056c-4a47-b8c4-67bfb4a2a64e.png)
![image](https://user-images.githubusercontent.com/36940685/196981471-85f3324c-3b47-45f7-9e0f-3e3ec7d8b95a.png)

However, when I click on it, nothing happens, and I get the following error in the console: 

![image](https://user-images.githubusercontent.com/36940685/196981577-f22c8d10-25dd-4a36-915e-b6afd0ac66a3.png)

#### Steps to reproduce
1. Create an Empty Navigation Bar
2. Add a "Page Link with sub-links" block
3. Add a main page, a name, and then add at least one sub-page link
4. Save the navbar, use it in your website, click on the button that has appeared and see the error in the console

#### Expected behavior
A dropdown menu to appear, as opposed to an error.

#### Additional context
I am trying to make a simple dropdown menu in the navbar that contains links to other pages, with the "main" button ideally not doing anything except making the menu appear. This does not seem to be supported by the existing blocks, but even any kind of dropdown menu seems to be broken for me.

**Comment by Sreini** [comment]
I managed to fix the issue after a lot of searching around. The default base.html page only loads `bootstrap.min.js` and does not load `popper.js`, which is apparently a library bootstrap uses to create dropdowns. I changed it to load `bootstrap.bundle.min.js` and it works now. I am creating a PR to change this, as I feel its kind of user-unfriendly for something like this to not work out of the box.

Unless of course I'm completely misunderstanding the issue and I had to solve it in a different way
